test/utils/pickers/openPicker.ts

```diff
@@ -31,3 +31,20 @@ export const openPicker = (params: OpenPickerParams) => {
   fireEvent.click(target);
   return true;
 };
+
+export const openPickerAsync = async (user: MuiRenderResult['user'], params: OpenPickerParams) => {
```

Comment:
The original `openPicker` would sometimes throw `act` errors. Again, not sure if it would be best to make the original async instead.

Reply:
Eventually - yes, but if we want to minimize the changes, we can go with intermediate step of deprecating the old one and migrating at a slower pace. 👌